### ISSUE TITLE
Are we forced to use NativeAuthToken or is it a bug? 

### ISSUE THREAD

**Issue by juliancwirko** [multiversx/mx-sdk-js-web-wallet-provider#24]
**Problem**:
- I use a standard token in the app (not NativeAuth, just a string)
- I want to use a web wallet and xPortal app. Steps:
  - I click web wallet in my app
  - I am redirected
  - I use xPortal in Web wallet
  - I land in my app
  - I want to make a transaction
  - I am redirected to web wallet again
  - I get the message in the browser console: `Invalid loginToken. You may be trying to decode a nativeAuthToken. Try using decodeNativeAuthToken method instead.` nothing happens. I stay in the web wallet

**Question**:
Are we forced to use NativeAuthToken logic, or is it something else?

---
xPortal for Android v2.0.29
web wallet provider v2.4.0

**Comment by andreibancioiu** [comment]
Reproduced.

```
https://wallet.multiversx.com/hook/login?token=example-auth-token&callbackUrl=https://localhost:8080/index.html 
-> "Invalid loginToken. You may be trying to decode a nativeAuthToken. Try using decodeNativeAuthToken method instead"

https://devnet-wallet.multiversx.com/hook/login?token=example-auth-token&callbackUrl=https://localhost:8080/index.html 
-> no error

https://testnet-wallet.multiversx.com/hook/login?token=example-auth-token&callbackUrl=https://localhost:8080/index.html 
-> no error
```

Fix will be published on the Web Wallet.

Thank you :pray:

**Comment by arhtudormorar** [comment]
A fix was deployed today. Please re-check

**Comment by juliancwirko** [comment]
Yes, the error in the console is gone. 

The only thing left in my case is that when I do the first transaction after signing in using web wallet + xPortal, I only land in the web wallet. The popup with sign info doesn't show, but when I go back to my app (where I am already signed in) and do the same transaction once again, the popup shows up, and I can sign it in xPortal. I've also tried a hard refresh in my app after sign-in redirection and the same result. The first transaction redirection doesn't trigger the signing process, but the second try does.

---
Tested on the devnet.
xPortal for Android v2.0.29
web wallet provider v2.4.0
At: https://multiversx-nextjs-dapp.netlify.app/
